torch/csrc/jit/codegen/cuda/compute_at_map.h

```diff
@@ -151,6 +169,10 @@ class TORCH_CUDA_CU_API ComputeAtMap {
   //! Get the ID sets for a provided IdMappingMode
   const DisjointSets<IterDomain*>& getIdSets(IdMappingMode mode) const;
 
+  //! Returns the pre-allocated index variable integer used in
+  //!  the kir::ForLoop corresponding to the given IterDomain.
```

Comment:
Nit: Just mention in the comment only valid if the ID has a loop mapping.

Reply:
updated. Thanks.